packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -524,6 +522,8 @@ export async function getPageStaticInfo(params: {
           }
         }
       }
+    } else if (pageType === 'pages') {
+      extraConfig = { ...config }
```

Comment:
This will also include e.g. `amp: true` and other configurations that don't affect the configuration but affect bundling right? Is that intentional?